### PR TITLE
Added Kubernetes Dashboard button to Web UI

### PR DESCRIPTION
**Pull request by ilyakooo0**
- Added a `kubernetes_dashboard_url` variable to the octopod deployment

  If it is empty, then the generated config has the value `null` for the URL.
- Added a "View logs" button to the "Deployments" and "Deployment" pages. The buttons will be absent if the field in the config is `null`.
- Added troubleshooting section to FAQ.

https://user-images.githubusercontent.com/6209627/104732085-bc689180-574d-11eb-9052-d50287c4859b.mov

### CODE REVIEW

File: charts/octopod/templates/octopod-nginx-configmap.yaml [hunk 1]
```diff
@@ -16,5 +16,10 @@ data:
     {
       "app_url": "https://{{ .Values.app_domain }}",
       "ws_url": "wss://{{ .Values.ws_domain }}",
-      "app_auth": "Basic {{ .Values.basic_auth_token }}"
+      "app_auth": "Basic {{ .Values.basic_auth_token }}",
```
Comment (andor0) [inline charts/octopod/templates/octopod-nginx-configmap.yaml:19]: JSON doesn't support tail comma

File: docs/en/Octopod_deployment_guide.md [hunk 2]
```diff
@@ -345,6 +345,22 @@ base_domain="octo.example.com"
...
 
+#################################################
+# Kubernetes Dashboard Setup
+#################################################
+
+# You can supply a base url for your k8s dashboard you have set up
+# to view the cluster, in which the Octopod will create deployments.
+#
+# Example URL to supply:
+#   https://dashboard.stage.thebestagent.pro
```
Comment (andor0) [inline docs/en/Octopod_deployment_guide.md:356]: maybe https://dashboard.example.com is better?

File: octopod-frontend/src/Reflex/MultiEventWriter/Class.hs [hunk 3]
```diff
@@ -8,7 +8,7 @@ import           Data.Type.Equality
 import           Reflex.Dom
 
 
--- | Same as 'EventWriter' but without a fundep and can only have a concrete implementation by EventWriterT.
+-- | Same as 'EventWriter' but without a fundep .
```
Comment (andor0) [inline octopod-frontend/src/Reflex/MultiEventWriter/Class.hs:11]: redundant space before the point

**Review by andor0 — CHANGES_REQUESTED** [review]

File: charts/octopod/templates/octopod-nginx-configmap.yaml (hunk 1 above)
Comment (ilyakooo0) [reply charts/octopod/templates/octopod-nginx-configmap.yaml:19]: There is an `else` clause for the `if`. This comma is never a tail comma.

Comment (andor0) [reply charts/octopod/templates/octopod-nginx-configmap.yaml:19]: exactly.

File: octopod-frontend/src/Frontend/GHCJS.hs [hunk 4]
```diff
@@ -29,22 +29,18 @@ appAuthKey = "auth"
...
 -- websocket host and API authentication token.
 data ProjectConfig = ProjectConfig
-  { appUrl :: String  -- ^ API host.
-  , wsUrl :: String   -- ^ WS host.
-  , appAuth :: String -- ^ API authentication token.
+  { appUrl :: T.Text  -- ^ API host.
+  , wsUrl :: T.Text   -- ^ WS host.
+  , appAuth :: T.Text -- ^ API authentication token.
+  , kubernetesDashboardUrlTemplate :: Maybe T.Text
+  -- ^ A k8s dashboard url template to which the name of the deployment is appended.
```
Comment (andor0) [inline octopod-frontend/src/Frontend/GHCJS.hs:36]: can do translate `url` to uppercase

**Review by andor0 — APPROVED** [review]

**Pull request merged**